kbase-extension/static/kbase/js/widgets/function_output/kbaseAttributeMapping.js

```diff
@@ -171,14 +167,9 @@ define (
         loggedInCallback: function(event, auth) {
             if (this.token == null) {
                 this.token = auth.token;
-                this.render();
+                this.renderAndLoad();
```

Comment:
except for this, and removing a useless loggedOutCallback, the rest is linting / whitespace cleanup.